src/app/contexts/ServiceContext/__mocks__/index.jsx

```diff
@@ -7,10 +7,10 @@ import services from '#testHelpers/serviceConfigs';
  * async behavior in tests across the application when
  * using service contexts.
  */
-export const ServiceContext = React.createContext(services.default);
+export const ServiceContext = React.createContext({});
 
 export const ServiceContextProvider = ({ children, service, variant }) => (
-  <ServiceContext.Provider value={services[service][variant]}>
+  <ServiceContext.Provider value={services[service][variant || 'default']}>
```

Comment:
I'd put the default value of variant in the argument destructuring instead of here e.g.

```js
export const ServiceContextProvider = ({ children, service, variant = 'default' }) => (
```